Hardware/Hardware/Printer/ReceiptContent.swift

```diff
@@ -0,0 +1,11 @@
+/// Models the content of the receipt.
+/// To be fully implemented in https://github.com/woocommerce/woocommerce-ios/issues/3978
+public struct ReceiptContent {
+    public let lineItems: [ReceiptLineItem]
+    public let paymentIntent: PaymentIntent
```

Comment:
While working on #3979 I've realized everything would be easier if we don't pass the full PaymentIntent here, and we pass a different type that contains only what we need to render the receipt. 

That way we don't need to persist PaymentIntent to Storage. 

So this will change in #3979

Reply:
I don't see the required fields for EMV. I assume those come in a later PR?

Reply:
Those are declared as properties of PaymentIntent's Charge, modelling what Stripe does. In #4010 this model object flattens a bit, and the EMV fields are a property of ReceptDetails